frontend/src/pages/dashboard/components/tables/trucks-table/columns/columns.ts

```diff
@@ -0,0 +1,55 @@
+import { capitalizeFirstLetter } from '~/libs/helpers/helpers.js';
+import { type ColumnDef, type TruckEntity } from '~/libs/types/types.js';
+
+const columns: ColumnDef<TruckEntity>[] = [
+  {
+    header: 'Manufacturer',
+    accessorFn: (trucks): string => capitalizeFirstLetter(trucks.manufacturer),
+    footer: 'Manufacturer',
+    size: 200,
+    enableSorting: false,
```

Comment:
it will be more logical and clear to make default value for enableSorting = false, and for columns which need to be sorted add enableSorting = true